python-threatexchange/threatexchange/exchanges/impl/ncmec_api.py

```diff
@@ -39,14 +39,16 @@ class NCMECCheckpoint(
 
     # The biggest value of "to", and the next "from"
     get_entries_max_ts: int
+    next_fetch: t.Optional[str] = ""
+    last_fetch_time: t.Optional[int] = 0
 
     def get_progress_timestamp(self) -> t.Optional[int]:
         return self.get_entries_max_ts
 
     @classmethod
     def from_ncmec_fetch(cls, response: api.GetEntriesResponse) -> "NCMECCheckpoint":
         """Synthesizes a checkpoint from the API response"""
-        return cls(response.max_timestamp)
+        return cls(response.max_timestamp, response.next, int(time.time()))
```

Comment:
blocking: NCMEC's documentation asks us not to store these checkpoints long term, so let's respect that in this implementation. 

One way we could handle this is not to directly access this variable, but wrap it with a helper function that will return empty string if it's too old.

```
def get_resume_url_if_recent(self):
   if time.time() - self.last_fetch_time < SOME_CONSTANT_MAYBE_12_HOURS:
     return self.next_fetch
   return ""
```

Reply:
Could we use `is_stale` for this?

Reply:
added the func (and not is_stale)